### PR TITLE
Add unit tests for displaying complex types

### PR DESCRIPTION
**Pull request by rschlussel**
relates to https://github.com/facebook/presto/commit/8a97b144eac4131225cda815c0561a0b9982bfab

### CODE REVIEW

File: presto-main/src/test/java/com/facebook/presto/type/MapTypeTest.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+public class MapTypeTest
+{
+    @Test
+    public void testMapDisplayName()
+    {
+        MapType mapType = new MapType(BigintType.BIGINT, VarcharType.VARCHAR);
```
Comment (haozhun) [inline presto-main/src/test/java/com/facebook/presto/type/MapTypeTest.java]: static import

File: presto-main/src/test/java/com/facebook/presto/type/RowTypeTest.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import static com.facebook.presto.spi.type.DoubleType.DOUBLE;
+import static java.util.Arrays.asList;
+import static org.testng.Assert.assertEquals;
+
+public class RowTypeTest
+{
+    @Test
+    public void testRowDisplayName()
+    {
+        List<Type> types = asList(BOOLEAN, DOUBLE, new ArrayType(VarcharType.VARCHAR), new MapType(BOOLEAN, DOUBLE));
```
Comment (haozhun) [inline presto-main/src/test/java/com/facebook/presto/type/RowTypeTest.java]: static import

File: presto-main/src/test/java/com/facebook/presto/type/RowTypeTest.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        Optional<List<String>> names = Optional.of(asList("bool_col", "double_col", "array_col", "map_col"));
+        RowType row = new RowType(types, names);
+        assertEquals(row.getDisplayName(), "row(bool_col boolean, double_col double, array_col array<varchar>, map_col map<boolean, double>)");
```
Comment (haozhun) [inline presto-main/src/test/java/com/facebook/presto/type/RowTypeTest.java]: Also test row type without names.

**Comment by rschlussel** [comment]
Thanks for the review!  Addressed comments.

**Comment by martint** [comment]
Merged, thanks!

**Pull request closed without merging**